pkgs/by-name/wi/widevine-cdm/package.nix

```diff
@@ -1,3 +1,15 @@
-{ stdenv, callPackage }:
+{
+  lib,
+  stdenv,
+  callPackage,
+}:
 
-callPackage (./. + "/${stdenv.hostPlatform.system}.nix") { inherit stdenv; }
+let
+  targets = lib.listToAttrs (
+    lib.map (name: lib.nameValuePair name (./. + "/${name}.nix")) [
+      "aarch64-linux"
+      "x86_64-linux"
+    ]
+  );
+in
+callPackage (targets."${stdenv.hostPlatform.config}" or targets.x86_64-linux) { inherit stdenv; }
```

Comment:
Shouldn't this still use `stdenv.hostPlatform.system`? At least for me the value of `stdenv.hostPlatform.config` is `"x86_64-unknown-linux-gnu"`, not `"x86_64-linux"`.

Reply:
Yes, that was a typo